### ISSUE TITLE
Unable to extract Azure Guest VM metrics 

### ISSUE THREAD

**Issue by navya-goli** [Azure/azure-sdk-for-python#995]
Hi 

I want to extract Azure Guest Os metrics using "Azure Monitor REST API" .

I enabled Diagnostics in azure portal.

When I tried to query the API using the GET request provided in the below link:

https://msdn.microsoft.com/library/mt743621.aspx

It is displaying the metrics of Host Vm only.

Please provide me assistance in getting GUEST OS METRICS using API queries.

**Comment by lmazuel** [comment]
Hi @navya-goli 
Let me check with the Monitor team.
Thanks,

**Comment by lmazuel** [comment]
Got an answer from the team. Unfortunately today, there is no REST API to support retrieving guest OS metrics.  You can configure guest agent (WAD/LAD) to collect performance counters to their storage account and then use  Azure storage APIs to retrieve the data.

**Comment by navya-goli** [comment]
Can you provide me the  assistance to configure guest agent WAD/LAD to move their performance counters to storage account.

Thanks

**Comment by shivab90** [comment]
Hi 
I want to extract Azure Guest Os metrics using "Azure Monitor Poweshell.
I enabled Diagnostics in azure portal.
I have configured guest agent (WAD/LAD) to collect performance counters to their storage account, I have tried the below command to retrieve the data

Get-AzureTableEntity -TableName $tableName -StorageAccountName $ComputerInventory_StorageAccountName -StorageAccountAccessKey $StorageAccountAccessKey  -QueryString $querystring -ConvertDateTimeFields $true -GetAll $true -Verbose

Can anyone provide me the solution to resolve this below error.

ConvertFrom-JSON : Cannot convert the JSON string because a dictionary that was converted from the string contains the duplicated keys 'Timestamp' and 'TIMESTAMP'

Thanks
